evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -0,0 +1,38 @@
+from jina.executors.evaluators.text import BaseTextEvaluator
+
+class BleuEvaluator(BaseTextEvaluator):
+    """
+    :class:`BleuEvaluator`Bilingual Evaluation Understudy Score. 
+    Evaluates the generated sentence (actual) against a desired sentence. 
+    It will use the Bleu on NLTK package.
+    A perfect match will score 1.0 and a perfect unmatch will score 0.0
+    """
+    
+    def get_nltk_bleu_score(self, desired, actual):
+        import nltk.translate.bleu_score as bleu    
+        from nltk.translate.bleu_score import SmoothingFunction
+
+        return bleu.sentence_bleu(desired, actual, smoothing_function=SmoothingFunction().method4)
```

Comment:
according to the `README` this `method4` should only be used if `n-grams` of less than `4`? Shouldn't this be an option as input?

Reply:
To have an input of the` n-gram` size?

Reply:
to use or not the `smoothing_function`. It just feels incoherent the documentation message and the implementation, why are we forcing the usage of this smoothing function without making sure that the checks to activate it are valid?

I would just give the option to the user to use or not the smoothing function.

Reply:
u can ask for an input argument (`smoothing_method`) and keep a static map of `str`  -> `SmoothingFunction().method` or even use the `getattr(SmoothingFunction(), method)` (which is this reflexion we want to avoid, but maybe in this case may be justified)

Reply:
Yeah makes sense, I also just saw another thing that should improve the scoring, ill change some stuff

Reply:
What you think of how it's done now? I think it's maybe better to check the n-gram and then decide how to process it instead of getting input from the user?

Reply:
if there is a strong reference for this, I can agree